src/clarity-angular/tabs/vtabs-nav-group.ts

```diff
@@ -0,0 +1,25 @@
+
+import {Component} from "@angular/core";
+
+/**
+ * @class VTabsNavGroup
+ *
+ * @description
+ * A child component of VTabs that is a container for a group of nav links.
+ * An optional label can be passed in to give the group a title.
+ * clr-vtabs-nav-links are projected to the VTabsNavLink class.
+ *
+ * Style is set in the template, not dynamically.
+ */
+@Component({
+    selector: "clr-vtabs-nav-group",
+    template: `
+        <div class="nav-group">
+            <ng-content select="label"></ng-content>
```

Comment:
I think @youdz mentioned to me earlier that we are not querying by class or by tag name. Can you please check with him once? Also, what exactly is `label`? Is it the nav-group header? If yes, we should style it using a class no?